mos.yml

```diff
@@ -84,18 +84,18 @@ config_schema:
   - ["gdo.begin_move_time_ms", "i", 7000, {title: "Time allowed for movement to begin, ms"}]
   - ["gdo.pulse_time_ms", "i", 300, {title: "Output active time, ms"}]
 
-  - ["rgb", "o", {title: "RGB settings", abstract: true}]
-  - ["rgb.name", "s", "RGB", {title: "Accessory name"}]
-  - ["rgb.enable", "b", true, {title: "Enable this light in the accessory"}]
-  - ["rgb.in_mode", "i", 1, {title: "-1 - Absent, 0 - Momentary, 1 - Toggle, 2 - Edge, 3 - Detached"}]
-  - ["rgb.in_inverted", "b", false, {title: "Invert input, set to true for normally closed input"}]
-  - ["rgb.state", "b", false, {title: "State of the light"}]
-  - ["rgb.hue", "i", 0, {title: "Hue of the light"}]
-  - ["rgb.brightness", "i", 100, {title: "Brightness of the light"}]
-  - ["rgb.saturation", "i", 100, {title: "Saturation of the light"}]
-  - ["rgb.initial_state", "i", 3, {title: "Initial state on power-on: 0 - off, 1 - on, 2 - restore last state, 3 - matches input if in toggle mode, otherwise off"}]
-  - ["rgb.auto_off", "b", false, {title: "Whether the switch should automatically turn OFF after turning ON"}]
-  - ["rgb.auto_off_delay", "d", 0, {title: "Delay for automatically turning OFF, in seconds"}]
+  - ["lb", "o", {title: "Light bulb settings", abstract: true}]
```

Comment:
I decided to name it "lb" for light bulb to be compatible for upcoming changes for 4 chanel support and support for vintage, duo and color bulb.